app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -364,12 +364,15 @@
   bottom: 2px;
   width: 111px;
 
-  a {
+  > a, > span.icon {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own line